src/service/resources-impl.js

```diff
@@ -332,6 +332,27 @@ export class Resources {
   }
 
   /**
+   * Returns a promise to the layoutBox for the element. If the element is
+   * resource-backed then makes use of the resource layoutBox, otherwise
+   * measures the element directly.
+   * @param {!Element} element
+   * @return {!Promise<!../layout-rect.LayoutRectDef>}
+   */
+  getElementLayoutBox(element) {
```

Comment:
Yup, Layers will be able to take any element and give you it's bounds (it'll almost be this exact method).